aks/cert-manager.tf

```diff
@@ -13,15 +13,10 @@ resource "null_resource" "cert_manager_setup" {
   }
 }
 
-data "helm_repository" "jetstack" {
-  name = "jetstack"
-  url  = "https://charts.jetstack.io"
-}
-
 resource "helm_release" "cert-manager" {
   name       = "cert-manager"
-  repository = data.helm_repository.jetstack.metadata.0.name
-  chart      = "jetstack/cert-manager"
+  repository = "https://charts.jetstack.io"
```

Comment:
Why not use the data source for the `helm_repository`? In this case, it probably doesn't matter much but I can see the data sources for the helm repositories being declared in a separate `.tf` file and only changing them rather than change the repo url directly here.